.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 
 node_js:
-  - 0.10
```

Comment:
could this be updated to follow what hapi is [doing now](https://github.com/hapijs/hapi/commit/e42ed26be1784977a2fd0706136c08eb413da5b7)?